src/main/java/org/opensearch/security/auth/UserInjector.java

```diff
@@ -63,13 +66,18 @@ public UserInjector(Settings settings, ThreadPool threadPool, AuditLog auditLog,
 
     }
 
-    static class InjectedUser extends User {
+    public static class InjectedUser extends User {
```

Comment:
Why do we need to expand this from package private?

Reply:
To register it as a `writeable` in Base64CustomHelper::registerAllWriteables.